packages/icons/src/svg/download.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M12 16.575c-.133 0-.258-.02-.375-.063a.877.877 0 0 1-.325-.212l-4.6-4.6a.948.948 0 0 1-.275-.7c0-.283.092-.517.275-.7.183-.183.42-.28.712-.288.292-.008.53.08.713.263L11 13.15V4c0-.283.096-.52.287-.712A.968.968 0 0 1 12 3c.283 0 .52.096.713.288.191.191.287.429.287.712v9.15l2.875-2.875c.183-.183.42-.27.712-.263.292.009.53.105.713.288a.948.948 0 0 1 .275.7.948.948 0 0 1-.275.7l-4.6 4.6c-.1.1-.208.17-.325.212a1.106 1.106 0 0 1-.375.063ZM5 21c-.55 0-1.02-.196-1.413-.587A1.926 1.926 0 0 1 3 19v-2c0-.283.096-.52.288-.712A.967.967 0 0 1 4 16c.283 0 .52.096.713.288.191.191.287.429.287.712v2h14v-2c0-.283.096-.52.288-.712A.968.968 0 0 1 20 16c.283 0 .52.096.712.288.192.191.288.429.288.712v2c0 .55-.196 1.02-.587 1.413A1.926 1.926 0 0 1 19 21H5Z" fill="#132930"/>
```

Comment:
the code is a bit ugly, but I tried to match the output from supernova exporter, so we might not have any code changes when they fix the import.